scopes/ui-foundation/use-box/menu/menu.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useMemo, ReactNode } from 'react';
```

Comment:
can be removed from this PR 
Will be fixed in #6714 

https://github.com/teambit/bit/pull/6714/files#diff-da180d74b8fa499c7a01ddcb24322e320f17c1d02fd29d037f1a2d72d5251b45